### ISSUE TITLE
Installation documentation (+ dependency versions) not updated

### ISSUE THREAD

**Issue by akuji1993** [shakacode/react_on_rails#1036]
I'm trying to build a brand new react on rails project right now, following the instructions that you have posted in the README.md. Following those, I get a rails project with the react stuff in it, but the example page at http://localhost:3000/hello_world shows me that it's missing the GET method for that particular page. On searching for the Hello World Component in the project, I can also not find any trace of that. I found a hello_react Component, but http://localhost:3000/hello_react also returns nothing (seeing as how routes.rb is completely empty that makes sense). 

Can you provide me with the missing steps to get this or alter your instructions please?

**Comment by akuji1993** [comment]
I managed to get the app running using your youtube tutorial series on react-on-rails. However in that case, I also needed to change my node version (usually using 6.11.0 right now) to 7+ first, so it accepted and installed webpack-dev-server-3.0 and then had to revert to 6.9.0 to make it install another package and then I could run the rest of the commands and installs and open the page. 

It might be worth updating all of the dependencies to the same state, so this doesn't need to be done while installing anymore. Very tedious to do and "destroys the immersion" when I try to follow your video tutorial just to stop and correct another version mistake. Takes away from the "it's that simple", that you state in the end.
